src/components/transactions/ExecuteTxButton/index.tsx

```diff
@@ -26,9 +27,19 @@ const ExecuteTxButton = ({
   const txNonce = isMultisigExecutionInfo(txSummary.executionInfo) ? txSummary.executionInfo.nonce : undefined
   const isPending = useIsPending(txSummary.id)
   const { setSelectedTxId } = useContext(ReplaceTxHoverContext)
+  const safeSDK = getSafeSDK()
 
   const isNext = txNonce !== undefined && txNonce === safe.nonce
-  const isDisabled = !isNext || isPending
+  const isDisabled = !isNext || isPending || !safeSDK
+
+  const tooltipTitle =
+    isDisabled && !isNext
+      ? `Transaction ${safe.nonce} must be executed first`
+      : isPending
+      ? `There's a pending transaction`
```

Comment:
I would be more descriptive here, e.g.

```suggestion
      ? `You cannot execute while while a transaction is pending.`
```

Reply:
I was trying to juggle between a more complete description and a more succinct one given it's a tooltip. Said that are you still in favor of the longer one?

Reply:
Compared to the `title` on line 37, I think it needs more information. An alternative: "Pending transaction must first succeed".